### PR TITLE
Fix the ValueError for "AllowUsers" without value

### PR DESCRIPTION
**Pull request by chenlizhong**

### CODE REVIEW

File: insights/parsers/ssh.py [hunk 1]
```diff
@@ -83,7 +83,10 @@ class SshDConfig(Parser):
     def parse_content(self, content):
         self.lines = []
         for line in get_active_lines(content):
-            kw, val = [s.strip() for s in line.split(None, 1)]
+            try:
```
Comment (xiangce) [inline insights/parsers/ssh.py]: I don't think your logic is incorrect, however, I don't think `try ... catch ...` is a good way to solve this kind of problems that we can predict. How about e.g.:
```python
line_splits =[s.strip() for s in line.split(None, 1)]
kw, val = line_splits if len(line_splits) == 2 else line_splits[0], ''
```

Comment (chenlizhong) [reply insights/parsers/ssh.py]: ok

**Comment by bfahr** [comment]
@xiangce I've asked @chenlizhong to report this issue in insights-core and link it to this PR since its a parser error.

**Comment by chenlizhong** [comment]
Fix the issue: https://github.com/RedHatInsights/insights-core/issues/1034

**Comment by bfahr** [comment]
@chenlizhong thanks for the extra effort.

**Pull request merged**